constants/networks.go

```diff
@@ -0,0 +1,17 @@
+package constants
+
+const (
+	Hedera          = "Hedera"
+	Ethereum        = "Ethereum"
+	Polygon         = "Polygon"
+	EthereumChainId = 3
+	PolygonChainId  = 80001
+)
+
+var (
+	NetworkIdToName = map[uint64]string{
+		0:     Hedera,
+		3:     Ethereum,
+		80001: Polygon,
+	}
```

Comment:
If we have other networks, we will need to add them every time. Having the name in the `bridge.yml` might replace this